### PR TITLE
Added New Character - Block

### PR DESCRIPTION
**Pull request by RajKhare**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

# This pull request is related to issue(s), 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
Added new character - Block

### My steps to solve it,
New file : block.svg in /characters
updated code in the gui.js for the character to work.

### Links

https://github.com/RajKhare/labyrinth
  
  character

### CODE REVIEW

File: characters/Character.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (abishekvashok) [inline characters/Character.svg]: Can you rename this to something more specific pls? Maybe block.svg

Comment (RajKhare) [reply characters/Character.svg]: Sure! np

**Comment by niccokunzmann** [comment]
@RajKhare I would like to merge it but we have conflicts. Would you like to resolve those?

**Comment by RajKhare** [comment]
@niccokunzmann Sure! I shall be glad!

File: js/gui.js [hunk 2]
```diff
@@ -14,15 +15,17 @@ window.addEventListener("load", function()
...
     resolve({
-      "characters/sheep.svg": "Intelligent Sheep",
-      "characters/robo.svg": "Robo",
-      "characters/swammy.svg": "Swammy",
-      "characters/monster.svg": "Monster",
-      "characters/elephant.svg": "Elephant",
-      "characters/shadowman.svg": "Shadowman",
-      "characters/santa.svg": "Santa Clause",
-      "characters/detective.svg":"Agent X",
-      "characters/Geekomoji.svg": "Geekomoji",
```
Comment (niccokunzmann) [inline js/gui.js]: On both sides there are 9 characters. Please make it 10 on the right side or rename your pull request to "replace character xy with block character."

**Comment by RajKhare** [comment]
@niccokunzmann I hope that the conflict is fixed!

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**